src/index.html

```diff
@@ -9,10 +12,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The `href` attribute is pointing to a SCSS file (`styles/style.scss`). Browsers cannot directly interpret SCSS files. You need to compile your SCSS into a CSS file and link to that CSS file instead.